### PR TITLE
LRQA-58177 | master

### PR DESCRIPTION
**Pull request by brookedalton**
https://issues.liferay.com/browse/LRQA-58177

### CODE REVIEW

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:search

File: build-test-elasticsearch7.xml [hunk 1]
```diff
@@ -11,6 +11,22 @@ xpack.ml.enabled: false</echo>
...
+xpack.security.http.ssl.certificate: certs/elastic-certificates.crt
+xpack.security.http.ssl.certificate_authorities : [ "certs/ca.crt" ]
+xpack.security.http.ssl.enabled: true
+xpack.security.http.ssl.key: certs/elastic-certificates.key
+xpack.security.http.ssl.verification_mode: certificate
+xpack.security.transport.ssl.certificate: certs/elastic-certificates.crt
+xpack.security.transport.ssl.certificate_authorities : [ "certs/ca.crt" ]
+xpack.security.transport.ssl.enabled: true
+xpack.security.transport.ssl.verification_mode: certificate
+xpack.security.transport.ssl.key: certs/elastic-certificates.key</echo>
```
Comment (xbrianlee) [inline build-test-elasticsearch7.xml:26]: Sort

File: build-test-elasticsearch7.xml [hunk 2]
```diff
@@ -220,6 +288,175 @@ ${line.separator}CATALINA_OPTS="${CATALINA_OPTS} -Djavax.net.ssl.trustStore=${el
...
+
+						<exec dir="${elasticsearch.dir}/bin" executable="cmd" inputstring="liferay&#x0A;">
+							<arg value="-c" />
+							<arg value="elasticsearch-keystore add xpack.security.http.ssl.keystore.secure_password" />
+						</exec>
+
+						<exec dir="${elasticsearch.dir}/bin" executable="cmd" inputstring="liferay&#x0A;">
+							<arg value="-c" />
+							<arg value="elasticsearch-keystore add xpack.security.http.ssl.truststore.secure_password" />
+						</exec>
```
Comment (xbrianlee) [inline build-test-elasticsearch7.xml:439]: Should be `<arg value="/c" />`

File: build-test-elasticsearch7.xml [hunk 3]
```diff
@@ -1129,6 +1376,32 @@ xpack.monitoring.collection.enabled: true</echo>
...
+
+					<set-x-pack-user-passwords />
+
+					<start-x-pack-trial />
+
+					<sleep seconds="10" />
+
+					<antcall target="stop-elasticsearch" />
+
+					<enable-pki-properties />
```
Comment (xbrianlee) [inline build-test-elasticsearch7.xml:1392]: This needs a name that more clear, but after refactoring it'll be more clear how we should name our pki macrodefs

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:search

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:sf

**Pull request closed without merging**